web/themes/custom/jcc_components/includes/paragraph.inc

```diff
@@ -195,9 +195,15 @@ function jcc_components_paragraph_views_reference_location_map_list(array &$vari
 
   if ($map_list_attachment && !empty($display['#rows'])) {
     foreach ($display['#rows'][0]['#rows'] as $row) {
+
+      // Excluding country from address view to add it as additional variable.
+      $address_no_country = $row['#node']->field_address->view('default');
+      $address_no_country[0]['country']['#value'] = "";
```

Comment:
Are we certain that `$address_no_country` will never be empty? (ie, that `$row['#node']->field_address` always has something?) If so, this is ok, otherwise this would throw an undefined index exception.

Reply:
That's right, field_address is not a required field on Location content Type, it's supposed to be essential but could be empty. FIXED.